js/security-headers.js

```diff
@@ -0,0 +1,33 @@
+import env from "../config/env.generated.json";
+
+export default {
+  directives: {
+    defaultSrc: [
+      `'none'`
+    ],
+    scriptSrc: [
+      `'self'`,
+      `'unsafe-inline'`
+    ],
+    fontSrc: [
+      `'self'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    styleSrc: [
+      `'self'`,
+      `'unsafe-inline'`,
+      `https://code.cdn.mozilla.net`
+    ],
+    imgSrc: [
+      `'self'`,
+      `data:`,
+      `*`
```

Comment:
What if we set this to `http:` and `https:`? That way thumbnails from the web work, but no "mystery content" will work